databases/week2/task.sql

```diff
@@ -0,0 +1,42 @@
+--1. Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO
+    task (
+        title,
+        description,
+        created,
+        updated,
+        due_date,
+        status_id,
+        user_id
+    )
+VALUES (
+        'Read SQL Tutorial',
+        'Read and practice the SQL tutorial on W3Schools.',
+        NOW(),
+        NOW(),
+        '2023-12-31',
+        1,
+        1
+    );
+
+--2. Change the title of a taskUPDATE taskUPDATE task
+UPDATE task SET title = 'Wash and Iron Clothes' WHERE id = 1;
+
+--3. Change a task due date
+UPDATE task SET due_date = '2024-01-31' WHERE id = 2;
+
+--4. Change a task status
+UPDATE task SET status_id = 2 WHERE id = 3;
+
+--5. Mark a task as complete
+UPDATE task SET status_id = 3 WHERE id = 4;
```

Comment:
Try to not hardcode the status_id. Try to look for the id of status 'done' with a query.